src/main/resources/META-INF/lsp4ij-qute.xml

```diff
@@ -44,6 +44,7 @@
                        implementationClass="com.redhat.devtools.intellij.lsp4ij.operations.documentLink.LSPDocumentLinkAnnotator"/>
     <completion.confidence  language="HTML" order="before htmlText" implementationClass="com.redhat.devtools.intellij.lsp4ij.operations.completion.LSPCompletionConfidence" />
     <implicitUsageProvider implementation="com.redhat.devtools.intellij.qute.psi.internal.extensions.renarde.RenardeImplicitUsageProvider"/>
+    <typedHandler implementation="com.redhat.devtools.intellij.qute.editor.QuteTypedHandler"/>
```

Comment:
This declaration should be done in plugin.xml, because it is not linked to lsp

Reply:
done